src/infrastructure/entitys/hermeticity.ts

```diff
@@ -0,0 +1,34 @@
+import { Entity, Column, PrimaryColumn, PrimaryGeneratedColumn, CreateDateColumn } from 'typeorm';
+import { MPPHermeticity, HermeticityStatus } from '../../core/hermeticity';
+
+@Entity()
+export class Hermeticity {
```

Comment:
Same here with the name as the `Alert`, change it to something more indicative that it is related somehow to the sql repository or to the typeorm package.